### PR TITLE
replace query2 to query

### LINKED ISSUE #132: Content: "--query2" is not a valid option
When I use the query method, it fails.

I found this code in `GroupRestClient.java`
```
if (!Strings.isNullOrEmpty(queryRequest.getQuery())) {
            // param is "query2", not "query".
            query = UrlUtils.appendToUrlQuery(query, "query2=" + queryRequest.getQuery());
        }
```
Why is `query2` written here? If `query2` is used, an error will be reported, and `query` will not be wrong. The version of gerrit I use is 3.3.1

### PR DESCRIPTION
**Pull request by magaofei**
fix #132

### CODE REVIEW

File: src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java [hunk 1]
```diff
@@ -147,8 +147,7 @@ public QueryRequest query() {
     protected List<GroupInfo> query(QueryRequest queryRequest) throws RestApiException {
```
Comment (EFregnan) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java:147]: Could you please add a comment above the method declaration saying that this method does not support Gerrit versions lower than 3.2.0? 

Everything else looks good to me. Thank you for taking care of it!

**Comment by EFregnan** [comment]
Thank you for adding the comment. It looks good to me.

Let's wait for @uwolfer to see if the PR can be merged.

File: src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java [hunk 2]
```diff
@@ -144,11 +144,16 @@ public QueryRequest query() {
         };
     }
 
+    /**
+     * this method may does not support Gerrit versions lower than 3.2.0
+     * @param queryRequest {@link QueryRequest}
+     * @return {@link List<GroupInfo>}
+     * @throws {@link RestApiException}
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java:151]: Can you please remove these docs? they do not provide any additional value in my opinion.

**Review by uwolfer — CHANGES_REQUESTED** [review]
Thank you for your PR! Just one minor comment from my side, otherwise LGTM.

File: src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java (hunk 2 above)
Comment (magaofei) [reply src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java]: done

**Review by uwolfer — COMMENTED** [review]
sorry, that was a missunderstanding: I would like to keep the first line of the comment which is meaningful, but not the empty param and return docs. can you please re-add the first line?

**Comment by uwolfer** [comment]
Thank you @magaofei!

**Pull request merged**